news_app.js

```diff
@@ -0,0 +1,105 @@
+// const { doc } = require("prettier");
+
+// FETCH VARIABLES
+const topic = 'food';
+const source = `https://newsapi.org/v2/everything?q=${topic}&sortBy=popularity&pageSize=5&apiKey=b10b22f57fca436ca78791aa2c90a376`;
+
+// DOM VARIABLES
+const body = document.body;
+const news = document.querySelector('.news')
+
+
+// JS FETCH DATA
+const getData = async () => {
+    const response = await fetch(source);
+    if(response.status !== 200){
+        throw new Error('can not fetch the data');
+    };
+    const data = await response.json();
+    return data;
+};
+
```

Comment:
when working with fetch api i would create js module, if you do not like fetch api you can use axjos or rxjs, in line 15 you have number 200 in this scenerio it is quite obvious what this number is but... Try to always use some kind of enum or variable. So that after some time you do not have to think what this number was.